### PR TITLE
Fix - Merge master and cleanup

### PR DESCRIPTION
**Pull request by priesgo**
* Deleted unused versions `participants 1.0.4`, `metrics 1.1.0`, `reports 3.1.0` and `reports 4.1.0`
* Merging changes in master not in develop branch
* `builds.json` changed to support the used builds
* Added a mode `--only-prepare-sandbox` to `build2.py` so the models can be easily built using custom tools
* Old `build.py` is replaced

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+--------------------------
+* reports 4.0.0
+    - MD5 sum changed to a string
+    - Other files added to the cancer interpretation request
+    - New file types PARTITION, VARIANT_FREQUENCIES and COVERAGE
+* Build system and package versioning refactored
+* Python mock data factories
+
+
+[...]
```
Comment (greglever) [inline CHANGES.md]: can we please add the changes from `master`:
```
version 4.3.4 (20 Oct 2017)
-------------

* Minor changes:
    - Fixed erroneous import to participants_1_0_0 from participants migration

version 4.3.3 (18 Oct 2017)
-------------

* Minor changes:
    - Fixed erroneous import to participants_1_0_3_SNAPSHOT from participants migration
```

File: build2.py [hunk 2]
```diff
@@ -1,223 +0,0 @@
-#!/usr/bin/env python
```
Comment (greglever) [inline build2.py]: so `build.py` has now been replaced with what used to be `build2.py` ?

File: CHANGES.md (hunk 1 above)
Comment (priesgo) [reply CHANGES.md]: done

File: build2.py (hunk 2 above)
Comment (priesgo) [reply build2.py]: yes, and the old build.py is deleted.

File: setup.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
     "PyYAML==3.12",
 ]
 
-VERSION = "4.3.1"
+VERSION = "4.3.0"
```
Comment (greglever) [inline setup.py]: `master` is currently on 4.3.4
https://github.com/genomicsengland/GelReportModels/blob/b66324603773f6b42121b73a0ad5ab5236145e67/setup.py#L20

Do we want to make this 4.4.0 or something like that ?

Comment (priesgo) [reply setup.py]: aagggh I changed it in CHANGES.md and forgot about setup.py...
I thought about 4.3.5

Comment (priesgo) [reply setup.py]: done

**Pull request merged**